dozer-tests/README.md

```diff
@@ -53,6 +53,8 @@ Otherwise, if `error.json` is found, the framework expects `dozer` fails to star
 
 For all supported error expectation checks, see `expectations.rs`.
 
+In case of addition of new test case, be aware of non-colliding ports from the connection that you are creating for each test cases.
```

Comment:
This is not the correct place to document this issue. This README describes the end-to-end test framework, not the unit tests that start with `test_e2e_`

Reply:
Ah wrong place! will replace it